finished-work/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Apple Pie Recipe</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" href="main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+</head>
+<body>
+  <div id="recipe1">
+  <div class="recipe-picture">
+    <h1>Apple Pies</h1>
+  </div>
+    <div id="description" class="container">
+      <p>
+        This was my grandmother's apple pie recipe. I have never seen another one
+        quite like it. It will always be my favorite and has won me several first place
+        prizes in local competitions. I hope it becomes one of your favorites as well!
+      </p>
+      <img src="/Users/raphamontenegro/Documents/ironhack/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/recipe-info.png" class="img-recipe-info">
```

Comment:
Be careful when linking images and files. In this case the src is pointing to your computer. To fix this the src should look like this in your case ```../starter-code/apple-pie-starter-code/images/image-name-goes-here.png```